### ISSUE TITLE
Volume Spikes At First Second

### ISSUE THREAD

**Issue by Benji0TheFox** [mixxxdj/mixxx#12943]
### Bug Description

There's a bug where the replaygain makes the songs spike their volume at the first second. I'm not sure how to fix what the software caused.

### Version

2.4.0

### OS

Windows 11

**Comment by daschuer** [comment]
This can be https://github.com/mixxxdj/mixxx/issues/12289

Which file type is affected? 

What is the version of your mfplat.dll?

**Comment by Benji0TheFox** [comment]
M4A

10.0.22621.3085

**Comment by JoergAtGithub** [comment]
This seems to be a known Windows 11 bug, described in #12289. Please try to update to the very last Windows 11 update - does this help?

**Comment by Benji0TheFox** [comment]
That might have. I realized that when I downloaded M4As from the iTunes store, that it was the iTunes software glitching (or idk if it's on purpose or not) with the sound of the file if it's M4A directly from the store, but then when I downloaded it as MP3 before converting it into M4A with an audio converter, it helped fixed the volume problem I was having. So, I found out it was not a Mixxx issue, it was a digital file source issue.
